lib/recurly/resource.php

```diff
@@ -165,8 +165,7 @@ protected function getChangedAttributes($nested = false)
       if(!array_key_exists($attr, $this->_values)) { continue; }
 
       if(isset($this->_unsavedKeys[$attr]) ||
-         $nested && in_array($attr, $requiredAttributes) ||
-         (is_array($this->_values[$attr]) || $this->_values[$attr] instanceof ArrayAccess))
+         $nested && in_array($attr, $requiredAttributes))
```

Comment:
i don't think we can remove this can we? otherwise array additions won't be synced right?